src/ensembl/src/shared/components/uploaded-file/UploadedFile.scss

```diff
@@ -0,0 +1,27 @@
+@import 'src/styles/common';
+
+.uploadedFile {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 14px;
+  font-size: 12px;
+}
+
+.name {
+  flex-shrink: 1;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+
+.size {
+  flex: 0 0 auto;
+  white-space: nowrap;
+  font-weight: $light;
+}
+
+.delete {
+  flex: 0 0 auto;
+  margin-left: 6px;
```

Comment:
Please check if this margin is required. PR looks good otherwise.

Reply:
In Andrea's designs, the space between the file size and the close button is visibly larger than between the file size and the file name. I am not sure whether gap + margin-left, like I did here, is any better than just explicitly setting left and right margin on the fileSize element

<img src="https://user-images.githubusercontent.com/6834224/127892527-0c6dcae5-f3c8-4cb0-ae3f-9b767ee26732.png" width="400" />

Reply:
I checked with Andrea and updated the distances in this element. Changed the gap + margin-left combo to margin-left + margin-right on the fileSize element.